proactive-jupyter-kernel/kernel.py

```diff
@@ -191,10 +198,46 @@ def __trigger_pragma__(self, pragma_info):
             return self.__create_fork_environment_from_name__
         elif pragma_info['trigger'] == 'job_fork_env':
             return self.__create_job_fork_environment__
+        elif pragma_info['trigger'] == 'show_resource_manager':
+            return self.__show_resource_manager__
+        elif pragma_info['trigger'] == 'show_scheduling_portal':
+            return self.__show_scheduling_portal__
         else:
             raise PragmaError('Directive \'' + pragma_info['trigger'] + '\' not known.')
 
-    def draw_graph(self, input_data):
+    def __show_resource_manager__(self, input_data):
+        if 'host' in input_data:
+            url = os.path.join('https://', input_data['host'], 'rm')
+        else:
+            url = os.path.join('https://', self.proactive_config['proactive_server']['host'], 'rm')
+
+        if 'width' in input_data and 'height' in input_data:
+            data = IFrame(url, width=input_data['width'], height=input_data['height'])
+        else:
+            data = IFrame(url, width=1200, height=750)
+
+        content = {'data': {'text/html': data._repr_html_()},
+                   'metadata': {}
+                   }
+        self.send_response(self.iopub_socket, 'display_data', content)
+
+    def __show_scheduling_portal__(self, input_data):
```

Comment:
I wonder if we can factorize `__show_resource_manager__` and `__show_scheduling_portal__`

Reply:
Yes! Sure. I will do it before merging.